sdk/spring/spring-cloud-azure-integration-tests/test-resources/jdbc/mysql/test-resources.json

```diff
@@ -2,6 +2,12 @@
   "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
+    "testApplicationSecret": {
+      "type": "string"
+    },
+    "testApplicationOid": {
+      "type": "String"
+    },
```

Comment:
Why need this?

Reply:
Also, this is my concern, 
now these values are introduced in `test-resources/servicebus/test-resources.json`,
but when I test mysql's test only, I disabled all other ITs, and these values are not introduced in `/jdbc/mysql/test-resources.json`